setup.py

```diff
@@ -17,7 +17,7 @@
     license='MIT',
     packages=find_packages(exclude='tests'),
     package_data={'README': ['README.md']},
-    install_requires=['requests==2.2.1'],
+    install_requires=['requests>=2.2.1'],
```

Comment:
Why do we want this change? 
I was under the impression that its good practice to lock the package dependencies. Having it marked to "anything greater" may cause some future push to break that might not be fully backwards compatible. Right?